src/main/java/org/folio/consortia/controller/TenantController.java

```diff
@@ -28,8 +28,8 @@ public ResponseEntity<TenantCollection> getTenants(UUID consortiumId, Integer of
   }
 
   @Override
-  public ResponseEntity<Tenant> saveTenant(UUID consortiumId, @Validated Tenant tenant) {
-    return ResponseEntity.status(CREATED).body(service.save(consortiumId, tenant));
+  public ResponseEntity<Tenant> saveTenant(UUID consortiumId, @Validated Tenant tenant, Boolean forceCreatePrimaryAff) {
```

Comment:
does it make sense to do re-run of primary affiliations when PUT is called?

Reply:
cc @SerhiiNosko

Reply:
Good catch, for me its better to move this flag to PUT endpoint as well, because it will simplify POST since we don't need checks is tenant already saved etc. And contract of PUT more suitable for doing such operations